### DISCUSSION TITLE
Wild Web Developer version in Eclipse 2023-06

### DISCUSSION THREAD

**Discussion by oleosterhagen** [mickaelistria/eclipse-bluesky#1177] [Q&A] [answered]
It is still possible to update the version of Wild Web Developer for the upcoming Eclipse 2023-06? The developer builds for _2023-06 M1_ are currently using **1.0.2**.

**Comment by vrubezhny** [comment, marked as answer]
WWD 1.0.5 currently is submitted for 2023-06 and I hope it will be available in 2023-06 M2

**Reply by oleosterhagen** [reply]
I can see your [commit in SimRel](https://git.eclipse.org/c/simrel/org.eclipse.simrel.build.git/commit/wildwebdeveloper.aggrcon?id=8fa5a8fe3f3abaf86dec48d1c4edca9cd191018b). That is really helpful.

Thank you!
